clients/benchmarks/CMakeLists.txt

```diff
@@ -173,3 +173,5 @@ set_target_properties( rocblas-bench PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${PROJ
 add_dependencies( rocblas-bench rocblas-common )
 add_subdirectory ( ./perf_script )
 target_compile_options( rocblas-bench PRIVATE -DROCBLAS_BENCH )
+
+add_definitions( -DROCM_USE_FLOAT16 )
```

Comment:
Shouldn't this be `target_compile_definitions(rocblas-bench PRIVATE -DROCM_USE_FLOAT16)`?